Krzywicki_reflection_1.qmd

```diff
@@ -0,0 +1,56 @@
+
+This Course
+
+Please tell me a little about your prior experience with R ("I use it all the time", "I am still learning", "I've never used it before"). What do you like most about working with R? What are the biggest obstacles for you when working with R?
+
+I used R in Biometry and Eco Stats last year. Those were my first exposures to it. I would describe myself as "familiarish but not confident". My favorite thing about it so far is the powerful and flexible data visualization features. As someone who spent years making graphs in excel for reports, I really appreciate ggplot. 
+
+I think my biggest obstacle is syntax issues and remembering the steps I need to take to get things done. I could be a better annotator.
+
+Describe your experience working with spatial data and GIS (either in R, or ArcGIS, or QGIS etc.). How do you tend to use spatial data (e.g., making maps, designing field sampling, building statistical models)?
+
+I don't have much experience with spatial data! I would say the closest thing would be making maps in google earth of research site locations for funder reports from bird monitoring projects.
+
+For my dissertation, I anticipate potentially using spatial data for making maps or building statistical models.
+
+How do you feel about taking this course (e.g., neutral, positive, negative) and why (e.g., aspects you are worried about, or looking forward to)?
+
+Positive! A little apprehensive because I would not describe spatial thinking or statistics as my strong suit. But that's why I'm here to learn!
+
+
+
+After reviewing the syllabus, give some thought to your objectives for this course. Which parts of this course are going to be most useful to your research? What skills do you really want to focus on during the semester? Once you've given this some thought, craft 3-5 SMART objectives for you in this course. SMART objectives are Specific (e.g., I want to make maps in R), Measurable (e.g., I want to build 3 maps displaying different elements of a location in R), Achievable (e.g., I want to build 3 maps displaying different elements of a location using ggplot), Relevant (e.g., I want to build 3 maps displaying social and ecological elements of a location using ggplot), and Time-bound (e.g., I want to build 3 maps displaying social and ecological elements of a location using ggplot for the final project). Try to think about your overall objectives for the course so that we can use to assess your final project. Remember, these objectives should be based in where you are now and where you'd like to be when the course ends. If you're not sure how to answer this, schedule a meeting with me to chat about it.
+
+I want to build 1 map showing social and ecological elements along the flyway of a migratory bird.
+
+I want to make 1 map comparing human movement and bird movement across the US/Mexico border.
```

Comment:
These are great objectives! Could you come up with one more to hit the 3-5 mark? Maybe something related to annotating code or a specific visualization you'd like to try?